pydata_sphinx_theme/_templates/switcher.html

```diff
@@ -0,0 +1,53 @@
+<div class="dropdown">
+    <button type="button" class="btn btn-primary btn-sm navbar-btn dropdown-toggle" id="switcher_button" data-toggle="dropdown">
+        {{ switcher_prefix }}{{ release }}
```

Comment:
Were is `switcher_prefix` being defined? (or maybe a left over from a previous iteration)